Website/src/items/content/EdgeWebsite/Data/CMP Data/Venues.yml

```diff
@@ -50,3 +50,29 @@ Languages:
     - ID: "d9cf14b1-fa16-4ba6-9288-e8a174d4d522"
       Hint: __Updated
       Value: 20210825T114833Z
+  - Version: 2
```

Comment:
I am puzzled by this change. Was it accidental, voluntary, or automatic? What is the purpose? If it is not needed, please revert it. Thanks